open-reac/src/main/java/com/powsybl/openreac/parameters/output/network/ShuntCompensatorNetworkOutput.java

```diff
@@ -47,15 +49,18 @@ public int getExpectedColumns() {
     @Override
     protected void readLine(String[] tokens, StringToIntMapper<AmplSubset> stringToIntMapper) {
         String id = stringToIntMapper.getId(AmplSubset.SHUNT, Integer.parseInt(tokens[ID_COLUMN_INDEX]));
-        double b = readDouble(tokens[B_COLUMN_INDEX]);
-        String busId = stringToIntMapper.getId(AmplSubset.BUS, Integer.parseInt(tokens[BUS_COLUMN_INDEX]));
-        Boolean reconnect = null;
         ShuntCompensator shuntCompensator = network.getShuntCompensator(id);
-        if (busId != null && shuntCompensator != null && busId.equals(
-            shuntCompensator.getTerminal().getBusView().getConnectableBus().getId())) {
-            reconnect = true;
+        if (Objects.isNull(shuntCompensator)) {
+            modifications.add(new ShuntCompensatorModification(id, null, null));
```

Comment:
Is there an interest to add this modification when the `shuntCompensator` is `null` ?

Reply:
Indeed no, I've left it to be isofunctional with the previous implementation. I removed it. Thanks for the comment